README.rst

```diff
@@ -0,0 +1,18 @@
+Tornado Prometheus Exporter
+===========================
+
+Prometheus exporter for Tornado applications. Inspired by `Flask Prometheus Exporter <https://github.com/rycus86/prometheus_flask_exporter>`_.
+
+*This is early alpha. At least, tests, proper docs are missing. Thus, it's not published to Pypi yet.*
+
+
+Usage
+-----
+
+Instead of using ``tornado.web.Application`` as base class, use ``tornado_prometheus_exporter.Application``.
```

Comment:
Is it likely that other plugins would require the same? What would you do then?

Reply:
Create a common subclass. The alternative would be to provide a Mixin which would crap out static code analysis and be not more convenient because __everyone__ would have to create a subclass with the mixin first

Reply:
Hm. Let's see how this works out.